pos_event_sale/static/src/js/popups/EventTicketsPopup.js

```diff
@@ -54,6 +54,19 @@ odoo.define("pos_event_sale.EventTicketsPopup", function(require) {
             );
         },
 
+        getTicketFinalPrice: function(ticket) {
```

Comment:
```suggestion
        getTicketPriceReduce: function(ticket) {
```

For consistency with the backend field: [price_reduce](https://github.com/odoo/odoo/blob/44f7754af4fc82228f4f12d3f0b59a2cfe7ecab5/addons/event_sale/models/event.py#L83)